files/en-us/web/accessibility/aria/roles/meter_role/index.html

```diff
@@ -0,0 +1,85 @@
+---
+title: 'ARIA: meter role'
+slug: Web/Accessibility/ARIA/Roles/meter_role
+tags:
+  - ARIA
+  - Accessibility
+  - Meter
+  - Web Development
+---
+<p>The <code><a href="https://www.w3.org/TR/wai-aria-practices-1.2/#meter">meter</a></code> role is used to identify an element being used as a meter.</p>
+
+<h2 id="Description">Description</h2>
+
+<p>A meter is a graphical display of a numeric value within a defined range. For example, showing battery percentage. A meter is not appropriate for values that do not have a meaningful maximum limit. Meters should not be used to indicate progress (for example loading), this should be communicated with the <code>progress</code> role.</p>
```

Comment:
```suggestion
<p>A meter is a graphical display of a numeric value within a defined range. For example, showing battery percentage. A meter is not appropriate for values that do not have a meaningful maximum limit. Meters should not be used to indicate progress (for example loading), this should be communicated with the <code>progress</code> element.</p>
```

What do you think about recommending the `progress` element instead? I was thinking it'd be better to defer to HTML when there already is an element that has the same AT mappings under the hood.